tests/keras/backend/backend_test.py

```diff
@@ -1281,8 +1281,8 @@ def test_separable_conv(self,
                                           (input_depth, depth_multiplier))
         _, pointwise = parse_shape_or_val((1,) * len(kernel_shape) +
                                           (input_depth * depth_multiplier, 7))
-        y1 = KNP.separable_conv(x, depthwise, pointwise,
-                                padding=padding, data_format=data_format)
+        y1 = getattr(KNP, op)(x, depthwise, pointwise,
```

Comment:
Is this related to this PR?

Reply:
It is and for a very nice reason. Since we don't pollute the backend namespace, you can't use K.separable_conv, because it's not supposed to be public. Similarly to how people used to do `K.tf`. This change is necessary for the build to pass.